addons/docs/web-components/README.md

```diff
@@ -1,107 +0,0 @@
-<h1>Storybook Docs for Web Components</h1>
```

Comment:
Removed because it was outdated?

Reply:
really not sure

Reply:
@shilman should these files remain?

Reply:
Yeah there is still a bunch of addon-docs stuff to port into the official documentation. Please don't delete.